apps/wrf/install_wrf_omp_mvapich2.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+APP_NAME=wrf
+APP_VERSION=4.1.3
+SKU_TYPE=hb
+SHARED_APP=/apps
+MODULE_DIR=${SHARED_APP}/modulefiles/${SKU_TYPE}/${APP_NAME}
```

Comment:
I recommend that we make this consistent with our other installers that allow you to set environment variables before running the installer so that the user does not have to change the install scripts if they want to run on different SKUs or place the application in a different directory